test/contracts/stdlib_tests.aes

```diff
@@ -3,6 +3,7 @@
 include "Func.aes"
 include "Option.aes"
 include "List.aes"
+include "ListInternal.aes"
```

Comment:
Not necessary?

Reply:
I am not sure actually. It was added for some reason...

Reply:
It compiles just fine without it